docs/api/12_customer_usage/custome-usage.mdx

```diff
@@ -0,0 +1,132 @@
+---
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Get customer current usage
+
+## Route
+
+```curl title="GET"
+/api/v1/customers/:customer_id/current_usage
```

Comment:
Do we actually need `:` before `customer_id`?

Reply:
Yes, it's a standard way to show that this `customer_id` has to be replaced by a real customer id